app/router.js

```diff
@@ -6,7 +6,7 @@ const { healthcheck } = paths
 
 module.exports.paths = paths
 
-module.exports.bind = function (app) {
+module.exports.bind = (app) => {
```

Comment:
Small thing but if you only have one variable you don't need the brackets

So can be module.exports.bind = app => {...}

Reply:
Done